src/go/generate.sh

```diff
@@ -29,8 +35,16 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 SHADOW_REPO="${DIR}/../.gopath/src/github.com/googlecloudrobotics/core/src/go"
 
 export GOPATH="${DIR}/../.gopath"
-export GO111MODULE=off
-go get k8s.io/code-generator/...
+# from "release-1.15" onwards k8s repositories are using go modules
+export GO111MODULE=auto
+rm -Rf "${DIR}/../.gopath/src/k8s.io"
+mkdir -p "${DIR}/../.gopath/src/k8s.io"
+cd "${DIR}/../.gopath/src/k8s.io"
+git clone --branch $K8S_BRANCH $GIT_API
+git clone --branch $K8S_BRANCH $GIT_APIMACHINERY
+git clone --branch $K8S_BRANCH $GIT_CODEGENERATOR
```

Comment:
can you use
go get k8s.io/code-generator/...@release-1.15
see
https://github.com/kubernetes/client-go/blob/master/INSTALL.md#enabling-go-modules

Reply:
Great, that looks much nicer.
I changed it.